power_grid_model_c/power_grid_model/include/power_grid_model/container.hpp

```diff
@@ -228,11 +228,13 @@ class Container<RetrievableTypes<GettableTypes...>, StorageableTypes...> {
 
     // get item per type
     template <class GettableBaseType, class StorageableSubType>
-    requires std::derived_from<StorageableSubType, GettableBaseType> GettableBaseType& get_raw(Idx pos) {
+        requires std::derived_from<StorageableSubType, GettableBaseType>
+    GettableBaseType& get_raw(Idx pos) {
```

Comment:
probably, the formatter will complain. the way to fix this is to `activate` your `venv`, run `pre-commit install` and do `pre-commit run`. please run all your `git` commits from a terminal with `activate`d `venv` so that you can actually run it. for now, you can do `pre-commit run --all-files` (i think, please read the pre-commit docs to check)